dashboards/ruby_vm/main.json

```
@@ -0,0 +1,164 @@
+{
+  "metric_keys": [
+    "total_allocated_objects"
+  ],
+  "dashboard": {
+    "title": "Ruby (VM) metrics",
+    "description": "A dashboard showing GC and VM statistics for MRI.",
+    "visuals": [
+      {
+        "title": "Total Allocated Objects",
+        "description": "The cumulative number of allocated objects",
+        "line_label": "%name%",
+        "display": "LINE",
+        "format": "number",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "total_allocated_objects",
```

```suggestion
        "title": "Allocated Objects",
        "description": "The number of allocated objects between minutely measurements",
        "line_label": "%name%",
        "display": "LINE",
        "format": "number",
        "draw_null_as_zero": true,
        "metrics": [
          {
            "name": "allocated_objects",
```
I've made the suggestion in PR https://github.com/appsignal/appsignal-ruby/pull/863 to not store the totals, but the differences between measurements.

---

Makes sense!